ci-operator/step-registry/cucushift/installer/rehearse/aws/c2s/ipi/disconnected/private/provision/cucushift-installer-rehearse-aws-c2s-ipi-disconnected-private-provision-chain.yaml

```diff
@@ -21,6 +21,9 @@ chain:
     - ref: ipi-conf-manual-creds-remove-unnecessary-creds
     - ref: ipi-install-install-aws
     - ref: ipi-install-times-collection
+    - ref: mirror-images-by-oc-image-mirror
+    - ref: enable-qe-catalogsource-disconnected
```

Comment:
@jianzhangbjz I don't think that this is required for C2S workflow, since your step has mirrored all images into the registry server. @jianzhangbjz @anpingli can you help confirm this?

Reply:
No, this step doesn't mirror the catalog source and optional operators' images.

Reply:
@jianzhangbjz good to know, thanks.